### ISSUE TITLE
XAML objekt for NavigationItems

### ISSUE THREAD

**Issue by Rogardo** [spiegelp/MaterialDesignExtensions#104]
It would be nice to have the  possibility, to define the navigation objects in XAML without a list via ViewModel.

The code variant is quite cool if you want a dynamic list, but only for app navigation a straight forward XAML definition is more easy to handle.

**Comment by DaveCS1** [comment]
Yes, I agree. I am currently working with that control now and it would be very helpful to be able to define items in XAML.
